qiita_db/handlers/artifact.py

```diff
@@ -69,6 +69,46 @@ def get(self, artifact_id):
 
         self.write(response)
 
+    @authenticate_oauth
+    def patch(self, artifact_id):
+        """Patches the filepaths of the artifact
```

Comment:
The "PATCH" request will allow us to update the filepaths attribute of the artifact. Currently, I'm only allowing to add a new filepath, which should be the html-summary, by using `op = add` and `path = /html_summary/". However, this may change if we need more functionality to the patch request.